packages/core/admin/admin/src/components/NpsSurvey/tests/index.test.js

```diff
@@ -23,30 +23,41 @@ const setup = () =>
     },
   });
 
+const waitForTimers = () => act(() => jest.runAllTimers());
```

Comment:
i don't see any benefit in this abstraction except a lack of clarity as to whats happening, how come you think it's necessary? :)